packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1224,6 +1224,47 @@ class EditableText extends StatefulWidget {
   ///   works only with [TextInputType.emailAddress]. Make sure the input field has a
   ///   compatible [keyboardType]. Empirically, [TextInputType.name] works well
   ///   with many autofill hints that are predefined on iOS.
+  ///
+  /// ### Troubleshooting Autofill
+  ///
+  /// Platform autofill services rely on [autofillHints] to work. Make sure the
+  /// entries in [autofillHints] are supported by the target platform, and the
+  /// autofill service currently in use (the name of the service can typically
+  /// be found in your mobile device's system settings).
+  ///
+  /// #### Autofill UI refuses to show up when I tap on the text field?
```

Comment:
nit: this is not phrased as a question, so probably shouldn't have a question mark?

Reply:
removed "?"